### ISSUE TITLE
input: existing austrian app

### ISSUE THREAD

**Issue by exiledsurfer** [meitar/better-angels#43]
The ministry for women and education in austria already has an application for iphone and android that provides all of the services (and more) that buoy is working on, i suggest you take a look at it:

https://www.bmbf.gv.at/frauen/services/fem_help_app.html

it's available in english, german, bosnian, serbo-croatian, and turkish (for the user)

here's a video in english:

https://www.youtube.com/watch?v=KByvx6ViAyk&list=UU1rfX096sHtFtdYVt7GeSnQ

take a look at it :-)

**Comment by meitar** [comment]
So the fem:HELP app you linked to, at least from the video demonstration I saw, is actually extremely different from Buoy.

In brief, Buoy is a community-driven emergency dispatch tool. This means it is designed to connect a person suffering domestic violence with other community members (trusted peers). In contrast, fem:HELP offers a collection of pre-compiled resources offered by institutions such as phone number hotlines, and shelters. fem:HELP is lacking the core ability that Buoy provides, which is to immediately notify your friends of the emergency, and then provide an interface *for your friends* (called "responders" in Buoy) to coordinate an immediate physical response to your alert.

Buoy accomplishes this task by sending an SMS/email/other alert to the people you have previously told the app that you trust. It then automatically creates an incident-specific (cryptographically unique) chat room that all of your responders can enter to coordinate a response. This, too, is a feature fem:HELP is lacking.

Another important note is that fem:HELP's resource listings are centrally managed. Like fem:HELP, Buoy also has the capacity to inform the user of local shelters, hotline numbers, and other legal aid resources. However, in Buoy, this information is as easy to customize for a specific Buoy instance as writing a WordPress post, and each installation of the Buoy plugin can provide *customized* resource information based on the location of its deployment or use case. Finally, note that Buoy already goes one step further than fem:HELP in that all of this information is overlaid onto a live-updating map. See also #18 for more details about the development of this particular feature with respect to physical-world resources such as shelters and food pantries, and so on.

Finally, note that unlike fem:HELP, Buoy currently is *not* designed to support the evidence-gathering process (documenting injuries, etc.). This is because there are already many apps that do this, and that do this far better than even fem:HELP. [Camera V, sponsored by WITNESS and produced by The Guardian Project come to mind](https://guardianproject.info/apps/camerav/). We see no need to reinvent the wheel.

In short, I appreciate being informed about fem:HELP, but I am left feeling mostly disappointed by it. It also seems that I have more work to do to distinguish Better Angels/Buoy from these other apps. Your input in how I can do this, once you become more familiar with what Buoy is working towards and how, would be appreciated! :)

FYI, I'll leave this issue open for a while so that others can see it in the issue tracker, but I will probably close it soon. In the mean time, pinging @unquietpirate and @letzgolilkittykat to make sure they see this, too.

**Comment by unquietpirate** [comment]
I watched the fem:HELP video and it seems relatively useful for what it is (a curated search engine for domestic violence resources), but it has different goals than Bouy does.

Also, it's a small point but one that's very important to me as a queer person, that Buoy is intentionally not gender-specific. Although the majority of domestic violence is perpetrated by men, victims of domestic violence can be people of any gender.

.

**Comment by exiledsurfer** [comment]
maymay:
the differences between fem:HELP and buoy were clear to me after i looked at the depository; i only wanted to show you what is already being used by the networks that i am involved with in austria - and this is tailored to the local perceptions for what those needs are. 

uqp:
fem:HELP was paid for by the austrian ministry for women and education; they have yet to move away from their gendered standpoint - if they ever will remains to be seen. 

----
in any case, i will continue to familiarize myself with the plugin and keep tabs on the progress, and give input where it seems meaningful and constructive.

**Comment by meitar** [comment]
> i only wanted to show you what is already being used by the networks that i am involved with in austria - and this is tailored to the local perceptions for what those needs are.

Yeah, I grok. :) I was disappointed because I thought from your description that the effort on fem:HELP was very different from the kind of resources being developed by State and institutional funding in the US and that appears not to be the case.

Thanks for the input and please keep us updated if there any exciting developments that can be worked/integrated with.
